Build fails on Ubuntu 20.04

```
[ 37%] Building CXX object csg/src/tools/CMakeFiles/csg_gmxtopol.dir/csg_gmxtopol.cc.o
cd /home/votca/votca/build/csg/src/tools && /usr/lib/ccache/g++  -DBOOST_ALL_NO_LIB -DBOOST_PROGRAM_OPTIONS_DYN_LINK -I/home/votca/votca/csg/include -I/home/votca/votca/tools/include -I/home/votca/votca/build/tools/include -isystem /usr/include/eigen3  -O2 --coverage -Wall -Wextra -Wpedantic -Wshadow -Wconversion -Werror   -std=c++14 -o CMakeFiles/csg_gmxtopol.dir/csg_gmxtopol.cc.o -c /home/votca/votca/csg/src/tools/csg_gmxtopol.cc
[91mIn file included from /home/votca/votca/xtp/include/votca/xtp/checkpointreader.h:27,
                 from /home/votca/votca/xtp/include/votca/xtp/checkpoint.h:23,
                 from /home/votca/votca/xtp/include/votca/xtp/qmatom.h:25,
                 from /home/votca/votca/xtp/include/votca/xtp/aoshell.h:28,
                 from /home/votca/votca/xtp/include/votca/xtp/aobasis.h:24,
                 from /home/votca/votca/xtp/src/libxtp/aobasis.cc:19:
/home/votca/votca/xtp/include/votca/xtp/checkpointtable.h: In constructor 'votca::xtp::CptTable::CptTable(const string&, const size_t&, const CptLoc&)':
/home/votca/votca/xtp/include/votca/xtp/checkpointtable.h:51:38: error: implicitly-declared 'H5::DataSet& H5::DataSet::operator=(const H5::DataSet&)' is deprecated [-Werror=deprecated-copy]
```

Reproduce with:
```
docker run -it votca/buildenv:ubuntu /bin/bash
git clone --recursive -b stable https://github.com/votca/votca
cd votca
mkdir build
cd build
cmake .. -DBUILD_XTP=ON
make
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,9 @@
 For more detailed information about the changes see the history of the
 [repository](https://github.com/votca/xtp/commits/master).
 
+## Version 1.6.1 (released XX.04.20)
+* fix warnings on Ubuntu 20.04 (#438)
+
 ## Version 1.6 _SuperPelagia_ (released 17.04.20)
 * fix 32-bit build (#381, #380)
 * remove duplicated basissets (#384, #386, #387)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -47,10 +47,14 @@ endif()
 find_package(LIBXC REQUIRED)
 find_package(HDF5 1.8 REQUIRED COMPONENTS CXX)
 
-if (${HDF5_VERSION} GREATER 1.8)
+if(HDF5_VERSION VERSION_GREATER 1.8)
   message(WARNING "HDF5 will be used such that it is compatible with version 1.8.")
 endif()
 
+# https://github.com/votca/xtp/issues/436, hdf5-1.10.4 generates a implicitly-declared operator warning
+if(HDF5_VERSION VERSION_EQUAL 1.10.4)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-deprecated-copy")
+endif()
 
 option(BUILD_SHARED_LIBS "Build shared libs" ON)
 include(GNUInstallDirs)

